k9mail-library/src/main/java/com/fsck/k9/mail/power/TracingPowerManager.java

```diff
@@ -25,7 +23,7 @@ public static synchronized TracingPowerManager getPowerManager(Context context)
         Context appContext = context.getApplicationContext();
         if (tracingPowerManager == null) {
             if (K9MailLib.isDebug()) {
```

Comment:
The way we did it in k9 itself was to plant/uproot the log tree in the places where the setting is set (and on startup).